vars/rollingUpgradePipeline.groovy

```diff
@@ -15,6 +15,12 @@ def call(Map pipelineParams) {
             SCT_GCE_PROJECT = "${params.gce_project}"
         }
         parameters {
+            string(defaultValue: '',
```

Comment:
What happens when the value is empty string?

Reply:
Not much if the user provides a base version param. If neither base version or image params are given, the job will fail (without a verbose and clear error message). We probably should some validation logic more generally.

Reply:
it happens anyway with all rolling upgrade, regardless the new one being added by this PR...

Reply:
thnx